npm-shrinkwrap.json

```diff
@@ -3380,6 +3390,11 @@
       "from": "extend@3.0.1",
       "resolved": "https://registry.npmjs.org/extend/-/extend-3.0.1.tgz"
     },
+    "fxa-content-server-l10n": {
```

Comment:
fxa-content-server-l10n isn't in package.json, instead it's installed [as a postinstall script](https://github.com/mozilla/fxa-content-server/blob/issue-5188-max-event-offset/package.json#L7). I think this should not be a part of npm-shrinkwrap.json